llvm/utils/repo/repo.cmake

```diff
@@ -24,10 +24,57 @@ else ()
     set (utils_dir /usr/share/repo)
 endif ()
 
-SET (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -O0 -fno-exceptions -fno-rtti" CACHE STRING "Default C Flags Debug")
-SET (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -O0 -fno-exceptions -fno-rtti" CACHE STRING "Default CXX Flags Debug")
-SET (CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -O3 -fno-exceptions -fno-rtti" CACHE STRING "Default C Flags Release")
-SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -fno-exceptions -fno-rtti" CACHE STRING "Default CXX Flags Release")
+# The user may specify the libcxxabi_include by giving
+# '-Dlibcxxabi_include:STRING=/path/to/clang/include' on the command line
+if (libcxxabi_include)
+	set (libcxxabi_include "${libcxxabi_include}")
+endif ()
+
+# The user may specify the exception and rtti by giving
+# '-Dexception_rtti:BOOL=Yes' on the command line
```

Comment:
Is this something that we want to allow (given that we don't yet have exception support in the tools)?

Reply:
Fixed in the commit https://github.com/SNSystems/llvm-project-prepo/pull/145/commits/6c4f10ca7cb881e1330eb14bea461620683715d0.